src/views/Search/index.tsx

```diff
@@ -41,18 +44,18 @@ const Search = ({ navigation, route }: Props) => {
     if (search) {
       fetchMoviesBySearch(search)
     }
-  }, [])
+  }, [route.params.search])
 
   const fetchMoviesBySearch = async (search: string): Promise<void> => {
     try {
       setIsDataFetching(true)
       const searchResponse = await fetch(
-        `https://api.betaseries.com/search/movies?key=${API_KEY}&text=${search}&limit=10`
+        `https://api.themoviedb.org/3/search/movie?api_key=${MOVIE_DB_API_KEY}&language=fr-FR&query=${search}&page=1`
```

Comment:
why not use movieAPIUrls here  ?
src/constants/index.ts

Reply:
Good point